Standards/scs-xxxx-v1-status-page-create-decision.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Status Page create decision
+type: Decision Record
+status: Draft
+track: Ops
+enhances: status-page-comparison.md
+---
+
+# Introduction
+
+Creating and maintaining IaaS is a complex task. Any kind of consumer (e.g. operators, cutsomers) can
+be supported by presenting the status of all possible parts of the
+serving infrastructure. Whether a service is not reachable or
+the used hardware is having an outage we want the consumers to be easily informed
+by using a "Status Page" application. The need for a "Status Page"
+came up early in Team OPS and the requirements a "Status Page" application 
+has to fulfill were defined and written down on 2022-06-02 as a 
+[MVP-0 epic](https://github.com/SovereignCloudStack/issues/issues/123).
+The upcoming research on existing solutions came to the conclusion that we want to
+create a new "Status Page" application.
+
+# Existing Applications
+
+Since we want to use as much as possible from existing projects and contribute to
+upstream projects to support the community with our efforts, it was a hard
+decision to create a new "Status Page" application.
+
+Before the decision was made some existing and known applications were tested
+and analyzed if they would fit to our use case. An overview of this
+comparsion can be found [here](https://github.com/SovereignCloudStack/Docs/blob/main/Decisions/status-page-comparison.md)
+While this is not not a complete list of all existing applications it did
+capture the most promising ones from the [awesome-status-page list](https://github.com/ivbeg/awesome-status-pages)
+in order to have base to decide upon.
+
+Work on an existing project only makes sense if the project is healthy OR can be
+brought into a healthy state. If upstream does not accept patches a fork is needed.
+The fork however only makes sense if the underlying technology is worth to be maintained.
+The possible candidates did not fullfill these - in the cases where
+contribution seemed possible the candidates looked abandoned and long existing CVEs weren't
+worked on.
+
+# Decision
+
+Based on the results the decision was made, that the effort is likely to be the same if
+we pick up an existing project and try to get it in shape for our use case. It was not
+100% clear if this would even be possible or if we still would have to maintain our
+own additional patches.
+
+So there will be a reference implementation that will match the requirements we have.
+In addition there will be an architecture design documentation. So if the reference
+implementation may not fit to you, it will be possible to create your own application.
+
+# Status Page Requirements and decisions from [Requirement discussion results](https://github.com/SovereignCloudStack/issues/files/8822531/20220602-status-page-scs-session.pdf)
```

Comment:
It's good to have .md (and not just the .pdf) with the discussions on the Design and Requirements.
However, I wonder whether putting this into a separate file and referencing it would be better to keep the ADR itself short in length.

To me, the key of this document is the evaluation of existing solutions (and where we could take them with reasonable effort) against the requirements (features & functionality, maturity, OSS health, etc.) and the decision that building upon existing code is likely significantly more hazzle than doing the work from scratch. 

If we want to include all information in the doc itself, then I would feel like we need the consolidated list of requirements AND the evaluation matrix included here. Or we reference both with a link.